### PR TITLE
scripts: Include hs_err files in the dump.

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

File: resources/collect-dump-logs.sh [hunk 1]
```diff
@@ -20,15 +20,16 @@ if [ ! -z $RUNNING ]; then
...
     sudo -u jicofo jmap -dump:live,format=b,file=${HEAP_FILE} ${PID}
-    tar zcvf jicofo-dumps-${STAMP}-${PID}.tgz ${THREADS_FILE} ${HEAP_FILE} /var/log/jitsi/jicofo.log
+    tar zcf jicofo-dumps-${STAMP}-${PID}.tgz ${THREADS_FILE} ${HEAP_FILE} /var/log/jitsi/jicofo.log /tmp/hs_err_*
     rm ${HEAP_FILE} ${THREADS_FILE}
 else
     ls $JAVA_HEAPDUMP_PATH >/dev/null 2>&1
     if [ $? -eq 0 ]; then
         echo "Jicofo not running, but previous heap dump found."
-        tar zcvf jicofo-dumps-${STAMP}-crash.tgz $JAVA_HEAPDUMP_PATH /var/log/jitsi/jicofo.log
+        tar zcf jicofo-dumps-${STAMP}-crash.tgz $JAVA_HEAPDUMP_PATH /var/log/jitsi/jicofo.log /tmp/hs_err_*
```
Comment (paweldomas) [inline resources/collect-dump-logs.sh]: was the 'v' option removed on purpose ?

Comment (bgrozev) [inline resources/collect-dump-logs.sh]: Yes, do you think we need it for any reason?

Comment (paweldomas) [inline resources/collect-dump-logs.sh]: I think we should ask Aaron or Damian if this output is captured somewhere

Comment (bgrozev) [inline resources/collect-dump-logs.sh]: OK, putting it back it.

**Pull request merged**